### PR TITLE
Depend on matplotlib-base to avoid pyqt dependency

### PR DESCRIPTION
**Pull request by maresb**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Basically, `matplotlib` = `matplotlib-base` + `pyqt`.  There are lots of packages, this one included, which depend on `matplotlib` and unnecessarily force conda to install qt, which is an extra 100MB.  For more background, see https://github.com/jupyter/docker-stacks/pull/896#issuecomment-515888808 .

### CODE REVIEW

**Comment by ahartikainen** [comment]
LGTM

**Comment by beckermr** [comment]
I made an issue and tagged you in it @maresb. If you have time to make a PR or even to simply comment there and add the rationale, that would be great!
